request_information.go

```diff
@@ -63,6 +63,11 @@ func (request *RequestInformation) GetUri() (*u.URL, error) {
 		request.SetUri(*uri)
 		return request.uri, nil
 	} else {
+		_, baseurlExists := request.PathParameters["baseurl"]
+		if strings.Contains(strings.ToLower(request.UrlTemplate), "{+baseurl}") && !baseurlExists {
```

Comment:
```suggestion
		if !baseurlExists && strings.Contains(strings.ToLower(request.UrlTemplate), "{+baseurl}") {
```

so we don't have to check the string if we already have a value